src/wasm/compat/mod.rs

```diff
@@ -424,4 +428,15 @@ impl TombWasm {
         // Ok
         Ok(mount)
     }
+
+    #[wasm_bindgen(js_name = receiveFile)]
+    pub async fn receive_file(&mut self, shared_file: WasmSharedFile) -> TombResult<Uint8Array> {
```

Comment:
receiving files shouldn't require auth- is taking a &mut self here appropriate? it seems like having this function here as opposed to mount.rs is the better choice, and it seems like we will need a `Client` to reconstruct via the remote blockstore.